app/src/CoreGameLogic/Feature/Spielzug/Aktion/CancelAllInsurancesForPlayerAktion.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Domain\CoreGameLogic\Feature\Spielzug\Aktion;
+
+use Domain\CoreGameLogic\EventStore\GameEvents;
+use Domain\CoreGameLogic\EventStore\GameEventsToPersist;
+use Domain\CoreGameLogic\Feature\Moneysheet\State\MoneySheetState;
+use Domain\CoreGameLogic\Feature\Spielzug\Aktion\Validator\HasPlayerANegativeBalanceValidator;
+use Domain\CoreGameLogic\Feature\Spielzug\Aktion\Validator\HasPlayerAnyInsuranceValidator;
+use Domain\CoreGameLogic\Feature\Spielzug\Aktion\Validator\IsPlayerNotInsolventValidator;
+use Domain\CoreGameLogic\Feature\Spielzug\Dto\AktionValidationResult;
+use Domain\CoreGameLogic\Feature\Spielzug\Event\InsuranceForPlayerWasCancelled;
+use Domain\CoreGameLogic\Feature\Spielzug\State\PlayerState;
+use Domain\CoreGameLogic\PlayerId;
+use Domain\Definitions\Card\Dto\ResourceChanges;
+use Domain\Definitions\Card\ValueObject\MoneyAmount;
+use Domain\Definitions\Insurance\InsuranceFinder;
+
+class CancelAllInsurancesForPlayerAktion extends Aktion
```

Comment:
Explizit benennen und am besten noch einen Kommentar drüber, wofür das gedacht ist, damit man das niemals außerhalb des Insolvenzkontextes benutzt (weil es ja Geld zurückgibt). Command und Event am besten auch anpassen und da auch Kommentare rein
```suggestion
class CancelAllInsurancesForPlayerBeforeInsolvenzAktion extends Aktion
```